bin/netjsonconfig

```diff
@@ -146,11 +146,9 @@ def print_output(output):
     # if file object, get bytes
     if hasattr(output, 'getvalue'):
         output = output.getvalue()
-    # python2: always print
     # python3: if string, just print
     if isinstance(output, six.string_types):
```

Comment:
I don't know the source of the confustion but, I have given you multiple hints about six.
There is the direct answer that I don't like giving.
"Six is a Python 2 and 3 compatibility library. It provides utility functions for smoothing over the differences between the Python versions with the goal of writing Python code that is compatible on both Python versions. "
- Source: Your submission on GCI, https://pypi.org/project/six/1.13.0/

How I am have different research task for you, do we need python2 compatibility if we are dropping python 2?